.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,47 @@
-# Summary
+# Summary [Required]
+<!-- Provide a concise description (2-4 sentences) of what changes this PR implements and why -->
 
-# Demo
+# Demo [Required for UI changes]
+<!-- Add screenshots or link to screen recording demonstrating the changes -->
+<!-- Include both light/dark mode and different device sizes if it matters -->
 
-Add screenshots or flow video
+## Release Notes
+<!-- 
+Release notes with appropriate tags will be included in automatically generated release notes.
+Use the following guidelines to determine if you need release notes:
 
-# Release notes
+1. Changes targeting feature branches: Generally NO release notes needed
+2. Changes targeting `develop` branch: Include release notes if merchants should be informed
+3. Any user-facing changes (UI, workflows, public API changes): Include release notes
+-->
 
-<fixed> 
-- Fixed a bug.
+### Use appropriate tags for different types of changes:
+<fixed>
+- Fixed [specific issue] that caused [specific impact]
 </fixed>
 
 <new>
-- Added new feature.
+- Added ability to [specific capability] which enables [specific benefit]
 </new>
 
-# Ticket
+<changed>
+- Modified [specific behavior] to now [specific new behavior]
+</changed>
 
+## Technical Information [Optional]
+<!-- Include any technical details that would help reviewers understand implementation choices -->
+
+## Testing Instructions [Optional]
+<!-- Step-by-step instructions for reviewers to verify your changes work as expected -->
+
+## Ticket [Required]
```

Comment:
Sometimes we add ticket number as `CIOS-000 ` for the task which has no youtack ticket. So, should we make this optional so that we don't have to add these ticket number 🤔 ?

Reply:
Yes, we can. Then we need to disable Github action that fails pipeline if ticket number is not filled.
Let's discuss this next week